odoo/addons/base/transpiler/utils/utils.py

```diff
@@ -0,0 +1,11 @@
+def convert_as(val):
```

Comment:
these utils are not really generic/usable. I think they just should be put in the transpiler file. Nothing wrong with a simple helper in a file